qip-0008.md

```diff
@@ -0,0 +1,234 @@
+3: else then
+4:   add 1 region under prime
+5:   add degree(zone) zones under the new region
+6: end if
+```
+
+#### Old Zones Wait To Send ETXs
+Before an existing chain can send transactions to one of the new chains, the
+following sequence of events must occur:
+1) first, the new chain must mine a prime block that gets accepted by the
+network
+1) then the old chain must mine a prime block committing to the new chain as an
+ancestor.
+
+Once these two conditions occur, the old chain may begin to process external
+transactions destined to the new chain. These conditions must be satisfied for
+any chain wishing to send to any of the newly added chains.
+
+#### Example
+< TBD example diagram >
+
+### Security Statement
+This specification works as long as miners record the uncles they witness. If
+miners choose not to record uncles, then the protocol cannot effectively
+measure uncled entropy. Therefore the incentive structure of the network should
+make sure that the dominant strategy for a miner is to record uncles, in lieu
+of any incentive they may have to prevent the network from expanding. This
+should not be difficult, but incentive mechanisms will be left for another QIP.
+
+### Choosing Expansion Parameters
+The above protocol for tree expansion is defined around a few adjustable
+parameters, given below:
+
+#### TREE_EXPANSION_FILTER_ALPHA
+This is the smoothing factor (range 0-1) used by each zone in its low-pass filter to gather
+a long running average of the zone's security efficiency score. Choosing a
+larger $\alpha$ will make the filter less responsive; the tree expansion
+algorithm will be less susceptible to short term variations in the efficiency
+score, but will take longer to decide to trigger an expansion when one becomes
+necessary.
+
+> Choose TREE_EXPANSION_FILTER_ALPHA = 0.9
+
+#### TREE_EXPANSION_THRESHOLD
+This is the threshold (range 0-1) above which the $f_{global}$ score will begin the tree
+expansion decision process. This threshold should be chosen high enough to not
+be easily triggered by minor changes in node operating behavior, but not so
+high that the security efficiency becomes unacceptably low.
+
+> Choose TREE_EXPANSION_THRESHOLD = 0.5
+
+#### TREE_EXPANSION_TRIGGER_WINDOW
+Once all chains have confirmed $f_{global}$ above `TREE_EXPANSION_THRESHOLD`,
+this is the number of consecutive _prime_ blocks that $f_{global}$ must remain above
+the threshold to confirm the decision to expand the tree.
+
+> Choose TREE_EXPANSION_TRIGGER_WINDOW = 144 prime blocks (apprx 24hrs at 10m prime block time)
+
+#### TREE_EXPANSION_WAIT_COUNT
+Once the network has confirmed the decision to expand the tree, this is the
+number of _prime_ blocks to wait until the expansion is activated. This should
+be chosen to give node operators some time to adjust their infrastructure, if
+needed, to account for the upcoming network change.
+
+> Choose TREE_EXPANSION_WAIT_COUNT = 1008 prime blocks (apprx 1 week at 10m prime block time)
```

Comment:
We need to add a "TREE_EXPANSION_STABILIZATION_WINDOW". The new zone should have X prime blocks prior to being able to conduct any transactions. This window should be prior to the window that "waiting for old chains to reference before sending ETXs"

Reply:
what will we do about the new genesis block after the expansion? this QIP doesnt talk about it, how does that happen?

Reply:
Added `TRANSACTION_WAIT_COUNT`